### PR TITLE
Global Styles: add more fonts to list

### PR DESCRIPTION
**Pull request by nosolosw**
This PR adds more fonts and pairings, as well as handling fonts that are deprecated.

**TODO:**

- [ ] Design for pairings controls.

## Testing instructions

Install the FSE plugin locally:

* `npm install`
* `npx lerna run build --scope='@automattic/full-site-editing'`
* Move the `<local-path-to-calypso>/apps/full-site-editing/full-site-editing-plugin/` under  `wp-content/plugins` in your local WordPress install.

Install a theme that uses Global Styles:

* Clone the [themes](https://github.com/automattic/themes) repo locally.
* `cd varia`
* `npm install`
* `npm run build`
* Move `<local-path-to-themes>/varia` under `wp-content/themes` in your local WordPress install.

Test:

* Activate the Full Site Editing plugin and the Varia theme
* Go to a post.
* Open the Global Styles sidebar and verify that fonts work.

### CODE REVIEW

**Comment by pablohoneyhoney** [comment]
Started this [Figma](https://www.figma.com/file/hLHKR0ZHW0zlUuznW4oLo7/Global-Fonts?node-id=1%3A25) to refine this [list](https://docs.google.com/spreadsheets/d/1NTwkQqkwj-ekNaO0eRD4Cmt3pFGUXswfh_tFDbt_Qw8/edit). Work in progress.

**Comment by davipontesblog** [comment]
I am closing this in favor of the current efforts with core GS. @nosolosw please reopen as needed!

**Comment by nosolosw** [comment]
Reopening, as this is still relevant to dotcom specifically.

File: apps/full-site-editing/full-site-editing-plugin/global-styles/class-global-styles.php [hunk 1]
```diff
@@ -90,25 +90,57 @@ class Global_Styles {
...
+		'Nunito',
 		'Open Sans',
+		'Overpass',
+		'Petit Formal Script',
 		'Playfair Display',
 		'Poppins',
-		'Roboto',
+		'PT Sans',
+		'Quattrocento',
+		'Raleway',
```
Comment (simison) [inline apps/full-site-editing/full-site-editing-plugin/global-styles/class-global-styles.php]: Adding only Raleway font at https://github.com/Automattic/wp-calypso/pull/40087

**Comment by pablohoneyhoney** [comment]
Sharing here [the font and pairings list](https://href.li/?https://docs.google.com/spreadsheets/d/1NTwkQqkwj-ekNaO0eRD4Cmt3pFGUXswfh_tFDbt_Qw8/edit#gid=715553751)
And [the Figma file](https://href.li/?https://www.figma.com/file/hLHKR0ZHW0zlUuznW4oLo7/Global-Fonts?node-id=1%3A20)

**New Pairings**
Cabin | Raleway 
Chivo | Open Sans 
Playfair Display | Fira Sans
Arvo | Montserrat 
Space Mono | Roboto 

Pairings-4

Pairings

Pairings-1

Pairings-2


Pairings-3

**Comment by nosolosw** [comment]
Echoing here what I've talked to @pablohoneyhoney so everyone is on the same page. Added a new task in the issue description as well.

As it currently stands, pairings need visual controls associated with them. So far, we've been using SVG with some art direction ([see](https://github.com/Automattic/wp-calypso/pull/38896)) --- we'll need to create new SVGs for the new pairings if we want to continue with that direction.

A different approach we discussed is to revamp the pairings controls so they scale better -- Pablo explored a few options here and we would need to settle on which one we want to implement.

**Comment by vindl** [comment]
@nosolosw is this PR still needed?

**Comment by nosolosw** [comment]
My understanding is that this PR needs more work before it is ready (we want to retire some fonts + adding some new font pairing controls). cc @pablohoneyhoney

Unfortunately, since this was prepared I was tasked with other things and couldn't make much progress here. I'm happy to assist in a small capacity, review, etc, but I'm sorry to say I can't work on this myself at the moment and for the upcoming weeks.

I guess we can close this until we find a new dev?

**Comment by vindl** [comment]
> I guess we can close this until we find a new dev?

I think that would be fine.

**Pull request closed without merging**